front/pages/api/v1/w/[wId]/spaces/[spaceId]/members.ts

```diff
@@ -0,0 +1,189 @@
+import { isLeft } from "fp-ts/lib/Either";
+import * as reporter from "io-ts-reporters";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { withPublicAPIAuthentication } from "@app/lib/api/auth_wrappers";
+import type { Authenticator } from "@app/lib/auth";
+import { DustError } from "@app/lib/error";
+import { SpaceResource } from "@app/lib/resources/space_resource";
+import { apiError } from "@app/logger/withlogging";
+import type { SpaceType, WithAPIErrorResponse } from "@app/types";
+import { isString, PatchSpaceMembersRequestBodySchema } from "@app/types";
+
+interface PatchSpaceMembersResponseBody {
+  space: SpaceType;
+}
+
+/**
+ * @swagger
+ * /api/v1/w/{wId}/spaces/{spaceId}/members:
+ *   patch:
+ *     summary: Change the members in a space.
+ *     description: Changes the list of members that have a specified space available to them.
+ *     tags:
+ *       - Spaces
+ *     security:
+ *       - BearerAuth: []
+ *     parameters:
+ *       - in: path
+ *         name: wId
+ *         required: true
+ *         description: Unique string identifier for the workspace
+ *         schema:
+ *           type: string
+ *       - in: path
+ *         name: spaceId
+ *         required: true
+ *         description: Unique string identifier for the space
+ *         schema:
+ *           type: string
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             oneOf:
+ *               - type: object
+ *                 properties:
+ *                   memberIds:
```

Comment:
We don't want to document this endpoint. I think we can do that by not providing a swagger for it?